src/main/java/de/leonhard/storage/LightningBuilder.java

```diff
@@ -77,45 +81,45 @@ public static LightningBuilder fromDirectory(@NonNull final File file) {
   // Adding out settings
   // ----------------------------------------------------------------------------------------------------
 
-  public LightningBuilder reloadCallback(@Nullable final Consumer<FlatFile> reloadConsumer) {
+  public @NotNull LightningBuilder reloadCallback(@Nullable final Consumer<FlatFile> reloadConsumer) {
```

Comment:
Returning this can never be null, because otherwise you would not get into this method without an NPE! You can remove those here.